### ISSUE TITLE
Physics Mod may be causing crashes with NVIDIA GPUs

### ISSUE THREAD

**Issue by IceBlossom** [haubna/PhysicsMod#737]
**Describe the bug**
After updating my mods recently, I have been getting a lot of crashes related to NVIDIA's OpenGL drivers. I asked for some help in the Sodium discord, and they thought the two problematic mods could've been Iris or PM.
I disabled PM and haven't had a crash.
NVIDIA Driver version is 537.34

**To Reproduce**
Practically any action, mainly in MCC Island, could cause the game to crash. Turning off/on Iris shaders, alt-tabbing, pausing the game, WALKING.

Some he_err_pid.log files from 9/17, as well as the one I had today
[hs_err_pid21544.log](https://github.com/haubna/PhysicsMod/files/12679050/hs_err_pid21544.log)
[hs_err_pid24564.log](https://github.com/haubna/PhysicsMod/files/12679052/hs_err_pid24564.log)
[hs_err_pid25628.log](https://github.com/haubna/PhysicsMod/files/12679054/hs_err_pid25628.log)
(9/20) 
[hs_err_pid7148.log](https://github.com/haubna/PhysicsMod/files/12679057/hs_err_pid7148.log)

Minecraft Version: 1.20.1
Physics Mod Version: 3.0.8

Why does NVIDIA's drivers have to screw this much stuff up, haha

**Comment by haubna** [comment]
Does this happen without Sodium + Iris and only Physics Mod installed? Which GPU do you have? Which shaderpack do you use (try using Complementary and see if it still happens)? Also the latest.log file could be useful.

**Comment by IceBlossom** [comment]
1. I haven't tried only Physics Mod, but I have tried just Sodium as well as Sodium + Iris, and those instances didn't have issues. I will try it when I get home from school today.
2. I have a 3060 Ti
3. I use Complementary Unbound with Euphoria Patches
4. I'll enable PM again when I get home and get a latest.log file.

**Comment by IceBlossom** [comment]
Oops, pressed the wrong button haha

**Comment by haubna** [comment]
Yeah please try without Sodium + Iris as well since Sodium does some very hacky stuff with the Nvidia drivers. Not sure if it effects this crash.

**Comment by NoNameTeam-NoNamer** [comment]
I also use NVIDIA's GPU,and I've instulled sodium + Iris,but the game runs normally.

**Comment by NoNameTeam-NoNamer** [comment]
Maybe beacause you havn't instull sodium extra?
